libs/ledger-live-common/src/families/evm/logic.ts

```diff
@@ -38,13 +39,36 @@ export const getEstimatedFees = (tx: EvmTransaction): BigNumber => {
   );
 };
 
+/**
+ * Helper returning the potential additional fees necessary for layer twos
+ * to settle the transaction on layer 1.
+ */
+export const getAdditionalLayer2Fees = async (
+  currency: CryptoCurrency,
+  transaction: EvmTransaction
+): Promise<BigNumber | undefined> => {
+  switch (currency.id) {
+    case "optimism":
+    case "optimism_goerli": {
+      const additionalFees = await getOptimismAdditionalFees(
+        currency,
+        transaction
+      );
+      return additionalFees;
+    }
+    default:
+      return;
+  }
+};
+
 /**
  * List of properties of a sub account that can be updated when 2 "identical" accounts are found
  */
 const updatableSubAccountProperties = [
   { name: "balance", isOps: false },
   { name: "spendableBalance", isOps: false },
   { name: "balanceHistoryCache", isOps: false },
+  { name: "swapHistory", iOps: false },
```

Comment:
```suggestion
  { name: "swapHistory", isOps: false },
```

Reply:
Nice catch ! Done and typed to avoid having the issue again 👍